Create netsdk1013.md (#21012)
dotnet/docs: docs/core/tools/sdk-errors/netsdk1013.md

---

> title: "NETSDK1013: The TargetFramework value '{0}' was not recognized. It may be misspelled. If not, then the TargetFrameworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly."

Reviewer: Titles are supposed to be limited to 65 characters, so I would include only the error text, omiting the propose solution text. -- https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-good-titles?branch=master

```suggestion
title: "NETSDK1013: The TargetFramework value was not recognized."
```

description: How to determine and set a valid TargetFramework value
author: marcpop
ms.topic: error-reference
ms.date: 10/9/2020
f1_keywords:
- NETSDK1013
---
# NETSDK1013: The TargetFramework value was not recognized

> **This article applies to:** ✔️ .NET 3.1.100 SDK and later versions

Reviewer: ```suggestion
**This article applies to:** ✔️ .NET 3.1.100 SDK and later versions

```

> The SDK tries to parse the values provided in the project file for <TargetFramework> or <TargetFrameworks> into a well known value.  If the value is not recognized, the `TargetFrameworkIdentifier` or `TargetFrameworkVersion` value may be set to an empty string or `Unsupported`. 

Reviewer: ```suggestion
The SDK tries to parse the values provided in the project file for `<TargetFramework>` or `<TargetFrameworks>` into a well known value.  If the value is not recognized, the `TargetFrameworkIdentifier` or `TargetFrameworkVersion` value may be set to an empty string or `Unsupported`.
```

> To resolve this, please check the spelling of your `TargetFramework` value from the list of [supported frameworks](https://docs.microsoft.com/en-us/dotnet/standard/frameworks).

Reviewer: ```suggestion
To resolve this, check the spelling of your `TargetFramework` value from the list of [supported frameworks](../../../standard/frameworks.md).
```

> You can also set the `TargetFrameworkIdentifier` and `TargetFrameworkVersion` properties directly in your project file.

Reviewer: ```suggestion
You can also set the `TargetFrameworkIdentifier` and `TargetFrameworkVersion` properties directly in your project file.

```
